core/modules/catalog/helpers/getProductGallery.ts

```diff
@@ -10,7 +10,7 @@ import Product from '@vue-storefront/core/modules/catalog/types/Product';
 export default function getProductGallery (product: Product) {
   if (product.type_id === 'configurable' && product.hasOwnProperty('configurable_children')) {
     if (!config.products.gallery.mergeConfigurableChildren && product.is_configured) {
-      return attributeImages(product)
+      return uniqBy(attributeImages(product).concat(getMediaGallery(product)), 'src')
```

Comment:
What about?:
```js
[
   ...attributeImages(product),
   ...getMediaGallery(product)
]
```

I think it might be much more readable

Reply:
@Fifciu I have done the changes.